Plugins/NativeCamera/Source/Apple/CameraDevice.mm

```diff
@@ -533,13 +540,12 @@ fragment float4 fragmentShader(RasterizerData in [[stage_in]],
 
             // Actually start the camera session.
             [implObj->m_impl->avCaptureSession startRunning];
-
             // To match the web implementation if the sensor is rotated into a portrait orientation then the width and height
             // of the video should be swapped
-            return implObj->m_impl->cameraTextureDelegate->VideoOrientation == AVCaptureVideoOrientationLandscapeLeft ||
-                implObj->m_impl->cameraTextureDelegate->VideoOrientation == AVCaptureVideoOrientationLandscapeRight ?
+            return implObj->m_impl->cameraTextureDelegate->VideoOrientation == VideoOrientationAngle::LandscapeLeft ||
+                implObj->m_impl->cameraTextureDelegate->VideoOrientation == VideoOrientationAngle::LandscapeRight ?
                 CameraDimensions{implObj->m_impl->cameraDimensions.width, implObj->m_impl->cameraDimensions.height} :
-                CameraDimensions{implObj->m_impl->cameraDimensions.width, implObj->m_impl->cameraDimensions.height};
+                CameraDimensions{implObj->m_impl->cameraDimensions.height, implObj->m_impl->cameraDimensions.width};
```

Comment:
This change makes me nervous since this code is used in some shipped products. How can we be sure this isn't going to break something, and that even after this change the behavior still "matches the web implementation" as mentioned in the comment? Were you able to verify this somehow?

Reply:
I'm wondering if it should be 'fixed' as it's not broken. I'm in favor of leaving as is and add a comment.

Reply:
Seems reasonable.

Reply:
I've reverted the fix.